### DISCUSSION TITLE
want to verify output

### DISCUSSION THREAD

**Discussion by HamzaAli195** [AammarTufail/python_ka_chilla2023#47] [Q&A] [answered]
i just wnat to know that why the output of 
<img width="251" alt="verify1" src="https://user-images.githubusercontent.com/121845773/211214735-19b67d75-0ff2-487b-acb6-a277de4cd53e.PNG">
weight is in float form as i convert it into int.

**Comment by Masoodullah909** [comment, marked as answer]
> i just wnat to know that why the output of <img alt="verify1" width="251" src="https://user-images.githubusercontent.com/121845773/211214735-19b67d75-0ff2-487b-acb6-a277de4cd53e.PNG"> weight is in float form as i convert it into int.

 The describe() method calculates the mean and standard deviation using float values, even if the values in the column have been rounded to integers. These statistical measures require decimal precision, so the output of the describe() method will always include float values.
